components/FilterForm/FilterForm.js

```diff
@@ -1,50 +1,143 @@
-import React from "react";
+import React, { useState } from "react";
```

Comment:
I don't think we need to import React here. 

You used to have to import React in every component, but that has not been necessary for a while now [(see blog post)](https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).

```suggestion
import { useState } from "react";
```